src/demos/calcite-tile-select.html

```diff
@@ -319,6 +319,34 @@ <h3>Basic Heading Only Radio</h3>
                       </calcite-tile-select>
                     </div>
 
+                    <h3>Full Width</h3>
+                    <div>
+                      <calcite-tile-select checked width="full" name="basic-heading-only-radio" value="one" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                      <calcite-tile-select width="full" name="basic-heading-only-radio" value="two" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                      <calcite-tile-select width="full" name="basic-heading-only-radio" value="three" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                      <calcite-tile-select width="full" name="basic-heading-only-radio" value="four" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                      <calcite-tile-select checked width="full" name="basic-heading-only-radio" value="five" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                      <calcite-tile-select width="full" name="basic-heading-only-radio" value="six" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                      <calcite-tile-select width="full" name="basic-heading-only-radio" value="seven" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                      <calcite-tile-select width="full" name="basic-heading-only-radio" value="eight" show-input="none"
+                        heading="Tile title lorem ipsum">
+                      </calcite-tile-select>
+                    </div>
+
```

Comment:
We should probably add full width demos for all other permutations, including left/right/none for the default, large visual and heading only displays for both radios and checkboxes.  I know that's a lot of extra markup but it will come in handy for screenshot testing down the line.